lib/hydra/works/models/concerns/generic_file_behavior.rb

```diff
@@ -0,0 +1,44 @@
+module Hydra::Works
+  module GenericFileBehavior
```

Comment:
This should be Hydra::Works::FileBehavior, not Hydra::Works::GenericFileBehavior.  Hydra::Works::GenericFile is a *Generic* implementation of a Hydra::Works::File.  Other work classes that people create will get the behaviors of a Hydra::Works::File by including Hydra::Works::FileBehavior.  Most of them will be *specific* implementations of Hydra::Works::File meaning that they are, by definition *not Generic*.